src/numerics/include/metaphysicl/dynamicsparsenumberbase_decl.h

```diff
@@ -392,6 +411,49 @@ DynamicSparseNumberBase_decl_std_unary(ceil)
 DynamicSparseNumberBase_decl_std_unary(floor)
 DynamicSparseNumberBase_decl_std_binary_union(fmod) // TODO: optimize this
 
+#if __cplusplus >= 201103L
+DynamicSparseNumberBase_decl_std_unary(llabs)
+DynamicSparseNumberBase_decl_std_unary(imaxabs)
+DynamicSparseNumberBase_decl_fl_unary(fabs)
+// DynamicSparseNumberBase_decl_fl_unary(exp)
```

Comment:
Why are these commented?

Reply:
Same reason as the ones here earlier, although I admit the explanatory comment (on line 387 in the new version) is getting farther and farther away as we add more functions...